### PR TITLE
Readme tweak

### PR DESCRIPTION
**Pull request by sam-lewis**

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/richard-stowe/ProblemSolving/pull/2" target="_blank">ReviewNinja

File: README.md [hunk 1]
```diff
@@ -1,2 +1,4 @@
 ProblemSolving
 ===============
+
+We solve problems!
```
Comment (richard-stowe) [inline README.md:4]: I am not happy with this line !resolve

Comment (sam-lewis) [inline README.md:4]: why not? !completed

Comment (richard-stowe) [inline README.md:4]: Oh sorry, its fine !fixed

**Pull request merged**